portfolio/src/main/webapp/script.js

```diff
@@ -13,17 +13,28 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random facts to the page.
  */
-function addRandomGreeting() {
-  const greetings =
+function addRandomFacts() {
+  const facts =
       ['\'Wait for it...legendary\'', '\'Talent wins games, but teamwork and intelligence wins championships\'', 
       'Fact: I love sports, I practiced Taekwondo for 10 years', '\'Suit up\'', 'Fact: I have 3 dogs'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const random = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  document.getElementById('random-fact-container').innerText = random;
+}
+
+/** Request content from server and add it to page */
+function getContentUsingArrowFunc() {
+  try{
+      fetch('/data')
+  .then(response => response.json())
+  .then((fruits) => document.getElementById('fetch-container').innerHTML = fruits);
```

Comment:
try{
      fetch('/data')
  .then(response => response.json())  **----> do you mean I need to put response.text() here? ** 
  .then((fruits) => document.getElementById('fetch-container').innerText = fruits); **and here instead of .innerHTML, innerText, right?**

Reply:
yes, use response.text() after you changed your servlet to return a string
innerHTML vs innerText doesn't matter in this case b/c you don't have any HTML, so either one will work